### ISSUE TITLE
Abnormal instance stop

### ISSUE THREAD

**Issue by nhet** [iobroker-community-adapters/ioBroker.ring#525]
**Describe the bug**  
Since yesterday, the ring adapter instance stops abnormal.

**Screenshots & Logfiles**  
![image](https://github.com/iobroker-community-adapters/ioBroker.ring/assets/13017349/5898ad57-ed86-4861-869e-5981b69527d1)

**Versions:**  
 - Adapter version: 3.3.0
 - node: v18.16.0
 - js-controller: 4.0.24
 - Operating system: rpi 4b rasbian OS

**Comment by Chris25122** [comment]
Same Issue here

**Comment by mark77-DE** [comment]
Same here
node v16.20.0
Adapter: 3.3.0
js-controller: 4.0.24

Even after fresh adapter installation, no change

**Comment by Hini2001** [comment]
Same here
node v16.18.0
Adapter: 3.2.6
js-controller: 4.0.23

**Comment by mymeyer** [comment]
Gleicher Fehler
node v18.16.0
Adapter: 3.3.
js-controller: 4.0.24

**Comment by LosFritzos** [comment]
Same here...
Adapter: 3.3.0
node: v18.16.0
js-controller: 4.0.24

**Comment by tomtaz01** [comment]
Hallo,
selbe Problem hier:
Adapter: 3.3.0
node: v16.20.0
js-controller: 4.0.24

Es ist sogar so, dass der javascript adapter aufgrund der Fehler grade mit abstürtzt. Was ist passiert?

**Comment by Cason70** [comment]
Hallo, selbes Problem!
Schon mehrfach erfolglos den Tokken erneuert.
Problem bleibt bestehen.

**Comment by kevinlieser** [comment]
Same problem here.

**Comment by tccflash** [comment]
Same here.

**Comment by exaware** [comment]
Same Issue here

Node.js: v18.16.0
Adapter: 3.2.2
js-controller: 4.0.24

**Comment by theimo1221** [comment]
Most likely Ring changed something in their API.
It might be fixed by https://github.com/dgreif/ring/commit/5a69b5df155a906f0ae8b62104177f591d7e7814 but I have to look into it after work

**Comment by Cason70** [comment]
Läuft wieder! 👍
Instance is running again! 👍
Thank you

**Comment by kevinlieser** [comment]
Yep, running again. But the User agent was already present in that file. So nothing really changed...?!

**Comment by coolerkerl** [comment]
thanks for fixing
